front/pages/w/[wId]/builder/data-sources/public-urls.tsx

```diff
@@ -83,53 +75,13 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
     .filter(isWebsite)
     .map((ds) => ds.toJSON());
 
-  const connectorIds = websiteDataSources
-    .filter((ds) => ds.connectorId !== null)
-    .map((ds) => ds.connectorId) as string[];
-
-  const connectorsAPI = new ConnectorsAPI(
-    config.getConnectorsAPIConfig(),
-    logger
-  );
-
-  const connectorsRes = await connectorsAPI.getConnectors(
-    "webcrawler",
-    connectorIds
-  );
-  if (connectorsRes.isErr()) {
-    throw new Error("Failed to fetch connectors");
-  }
-
-  const dataSourcesWithConnector = websiteDataSources.map((ds) => {
-    const connector = connectorsRes.value.find((c) => c.id === ds.connectorId);
-    if (!connector) {
-      logger.error(
-        {
-          panic: true, // This is a panic because we want to fix the data. This should never happen.
-          workspaceId: owner.sId,
-          connectorId: ds.connectorId,
-          dataSourceId: ds.sId,
-          connectorProvider: ds.connectorProvider,
-        },
-        "Connector not found while we still have a data source."
-      );
-      return null;
-    }
-    return {
-      ...ds,
-      connector,
-    };
-  });
-
-  const dataSources = removeNulls(dataSourcesWithConnector);
-
   return {
     props: {
       owner,
       subscription,
       plan,
       readOnly,
-      dataSources,
+      dataSources: websiteDataSources,
```

Comment:
Not sure I understand why we don't need the connector anymore, but trusting you on this one :+1:

Reply:
Ok we needed that for the sync chip but it has been removed in August here: https://github.com/dust-tt/dust/pull/6601/files

And we were still loading connectors.

Reply:
Adding a commit to fix and bring back the sync chip.